### PR TITLE
Export 2018 03 23 (Rocket 2.0)

### PR DESCRIPTION
**Pull request by cnevinc**
These are the new strings for Rocket 2.0 (updated)

### CODE REVIEW

File: locales/templates/app.pot [hunk 1]
```diff
@@ -349,6 +354,28 @@ msgid ""
...
+msgid "1. Multitask by opening %s at the same time!"
+msgstr ""
+
+msgctxt "first_run_upgrade_page_text1_highlight"
+msgid "multiple tabs"
+msgstr ""
+
+#, c-format, python-format
+msgctxt "first_run_upgrade_page_text2"
+msgid "2. Access your favorite sites quickly by %s"
```
Comment (Delphine) [inline locales/templates/app.pot]: I feel like we may need a localization note speficiying what  %s is here.

File: locales/templates/app.pot [hunk 2]
```diff
@@ -349,6 +354,28 @@ msgid ""
 "using data) or tap it to revisit the web page."
 msgstr ""
 
+msgctxt "first_run_upgrade_page_title"
+msgid "We heard your feedback!"
+msgstr ""
+
+#, c-format, python-format
+msgctxt "first_run_upgrade_page_text1"
+msgid "1. Multitask by opening %s at the same time!"
```
Comment (Delphine) [inline locales/templates/app.pot]: What's  %s? I feel like we need to specify this

File: locales/templates/app.pot (hunk 1 above)
Comment (Delphine) [reply locales/templates/app.pot]: (maybe it's just me. please just comment on that and I'll determine)

**Comment by mTwTm** [comment]
Sorry for not making that clear at the first place!

The %s here are placeholders for other strings, for instance in 1. Multitask by opening %s at the same time! the %s will be replaced by "multiple tabs". first_run_upgrade_page_text1 is paired with first_run_upgrade_page_text1_highlight, the same for first_run_upgrade_page_text2.

It is formatted this way to offer different text styles. The "multiple tab" will be highlighted in the sentence. I might be able to provide some visual example tomorrow when these are landed.

**Comment by mTwTm** [comment]
screenshot_20180324-111028

**Comment by mTwTm** [comment]
Here's the sample :)

**Comment by Delphine** [comment]
Thanks for explaining!
Thing is in some languages, you need to know what the placeholder corresponds to in order to know how to translate the rest of the sentence (I'm thinking for example that in French it's the case).

Could we get a localization note in those cases to explain what these correspond to?
thanks

**Pull request merged**